news/utils/news.py

```diff
@@ -29,6 +29,22 @@ def get_reporter(username):
         return None
 
 
+def get_subscriber(username):
```

Comment:
I think get_subscriber_by_username is a better name for this function

Reply:
sure